### PR TITLE
Bug 1133965 - Support Camera and CameraFocus buttons r=wilsonpage

### PR DESCRIPTION
**Pull request by lissyx**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[lissyx  (lissyx)](https://github.com/lissyx) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=c69723434e42)

**Comment by try-server-hook** [comment]
[lissyx  (lissyx)](https://github.com/lissyx) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=dd81361712e9)

**Comment by try-server-hook** [comment]
[lissyx  (lissyx)](https://github.com/lissyx) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=397ccec3e2fd)

File: apps/camera/js/controllers/camera.js [hunk 1]
```diff
@@ -87,9 +87,40 @@ CameraController.prototype.bindEvents = function() {
...
+
+CameraController.prototype.handleKeyUp = function(ev) {
+  debug('ev:' + ev.key);
+  console.error('Received: ' + ev.type + ' for ' + ev.key);
+  var k = ev.key.toLowerCase();
+  switch (k) {
+    default:
+      debug('Unhandled keyup: ' + k);
+  }
+};
```
Comment (wilsonpage) [inline apps/camera/js/controllers/camera.js]: Do we need this function?

**Comment by try-server-hook** [comment]
[lissyx  (lissyx)](https://github.com/lissyx) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=4467eea59e84)

**Comment by try-server-hook** [comment]
[lissyx  (lissyx)](https://github.com/lissyx) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=6deb3d557ae2)

**Comment by try-server-hook** [comment]
[lissyx  (lissyx)](https://github.com/lissyx) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=c76cb1bfadec)

**Comment by try-server-hook** [comment]
[lissyx  (lissyx)](https://github.com/lissyx) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=899dc565a80c)

**Comment by try-server-hook** [comment]
[lissyx  (lissyx)](https://github.com/lissyx) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=1f67bf0d2b26)

**Pull request merged**